.bazelrc

```diff
@@ -0,0 +1,15 @@
+# Copyright 2020 The TensorFlow Authors. All Rights Reserved.
```

Comment:
Also, let's add `.bazelversion`, similarly to what IREE does? https://github.com/iree-org/iree/blob/main/.bazelversion

Reply:
Done.